### DISCUSSION TITLE
Billing Code Type Version to be used for RC (Revenue Code)

### DISCUSSION THREAD

**Discussion by vishalpatel2** [CMSgov/price-transparency-guide#132] [Q&A] [answered]
Hello, 
We are looking for guidance on What type of **billing code type version** to use when reporting INN (In Network) FFS rates with Revenue Code (RC). 

We were reviewing this [thread](https://github.com/CMSgov/price-transparency-guide/discussions/9#discussioncomment-587290) 
however didn't get much insight on guideline. 

Should that be something like UB-04-YYYY (Where YYYY indicates current year?) assuming UB-04 is the adopted billing information version by NUBC for a given year?  We appreciate your assistance in advance. Thanks

@StoneCypher @shaselton-usds  @kpatel-cms @rvguha @tdfow

**Comment by tdfow** [comment, marked as answer]
We are planning to use CCYY format for the version indicates the current publication version, as I haven't seen any other guidance

**Reply by vishalpatel2** [reply]
Thanks for quick confirmation.

**Comment by ghost** [comment]
@vishalpatel2 We are taking the same approach as @tdfow in using the CCYY format.
